src/lib/sceneEndpoints.js

```diff
@@ -81,3 +81,18 @@ export const deleteScene = async (id, handleError) => {
         throw error;
     }
 };
+
+export const duplicateScene = async (id, handleError) => {
+    try {
+        const response = await httpPost(
+            baseEndpoint + `scene\\${id}\\duplicate`,
```

Comment:
Why the double \\\? I didn't know about this JS syntax

Reply:
![image](https://user-images.githubusercontent.com/13268990/124365429-ef2c1680-dc15-11eb-978b-3f85d06a5318.png)

Single \ is treated as an escape character with the ${id} characters